xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-docker/src/test/it/org/xwiki/flamingo/test/docker/LoginIT.java

```diff
@@ -43,16 +42,16 @@
  * @version $Id$
  * @since 2.3M1
  */
-@UITest(servletEngine = ServletEngine.EXTERNAL,
+@UITest(
     properties = {
         // The RightsManagerPlugin is needed to change rights in the UI
         "xwikiCfgPlugins=com.xpn.xwiki.plugin.rightsmanager.RightsManagerPlugin"
```

Comment:
Note that this one should be added on AllIT too if I'm correct, unless we want to only rely on the merge of properties now. ping @vmassol  I don't recall the latest status on that.

Reply:
I will merge the PR I think it should be ok for now since we have the merging of properties, I assume you ran the whole module docker-tests and it was ok @manuelleduc right?

Reply:
From https://dev.xwiki.org/xwiki/bin/view/Community/Testing/DockerTesting/ 
"When using nested tests, only configure any @UITest parameter in the nested tests themselves and not in the parent (e.g. at the AllIT level the @UITest annotation should be empty). This is because @UITest annotation parameters are automatically merged."

Reply:
Ah thanks :)